### ISSUE TITLE
Make "translations" and "translations of translations" different

### ISSUE THREAD

**Issue by RyckRichards** [Tatoeba/tatoeba2#2853]
I've been linking sentences on Tatoeba but sometimes, I don't notice that one of them was actually a translation of translation because they use the same color and font.

![image](https://user-images.githubusercontent.com/7658430/134361134-6e71ed96-7b87-40f4-9bb8-859c61062258.png)

Perhaps we could put the "translations of translations" (or translations) in bold or even better: separate them with a line.

**Comment by Yorwba** [comment]
> I've been linking sentences on Tatoeba but sometimes, I don't notice that one of them was actually a translation of translation because they use the same color and font.

Could you tell us a bit more about what happened? E.g. did you accidentally link/unlink the wrong sentence because of this?

**Comment by RyckRichards** [comment]
Sure. I accidentally linked and unliked sentences and I also didn't notice at first that some sentences were not linked yet and therefore was "marked" as "translations of translations"

**Comment by Yorwba** [comment]
When you expand the menu to link and unlink, direct and indirect translations get marked with different icons, right?
![Sentence menu expanded](https://user-images.githubusercontent.com/6555947/134426714-e18fea2a-4392-452a-84d1-c949ef38dd9e.png)
Are they too similar to each other?

**Comment by RyckRichards** [comment]
> Are they too similar to each other?

Yes. I was trying to find a sentence that only contains "translations of translations" (indirect translations) to show what I'm saying better.

I often read "Translations" and I go to the next batch of sentences and I rarely realize that them were actually indirect translations .

**Comment by Yorwba** [comment]
The reason I asked about the icons is that they're the one thing you have to look at when you're linking/unlinking, so they're the best place to make the difference clear. If the chain and slashed chain icons are too similar, maybe we could make the slashed chain red to emphasize that it's a kind of destructive action and make it easier to tell them apart?

As for not noticing that there are indirect translations to link in the first place, I don't think emphasizing the "translations of translations" label is going to work in the long term. At first you might pay more attention to it, but once you get used to it, it'll just turn into part of the background. (Personally, I think I recognize the indirect translations based on the visual break created by the label, not the label itself.) And if the list is very long, you might not be able to see the label at all.

So I think it's best to have something about each sentence that tells you whether it's a direct translation or not. Right now we have the black vs gray arrows, but that's probably too subtle. In the old design, the indirect translations are gray, but that makes them kind of hard to read.
